app/models/bar.py

```diff
@@ -27,6 +27,7 @@ class BarItem(db.Model):
     price: Column[float] = column(sa.Numeric(6, 2, asdecimal=False), nullable=False)
 
     is_alcohol: Column[bool] = column(sa.Boolean(), nullable=False)
+    alcohol_degree: Column[bool] = column(sa.Numeric(4, 2), nullable=True)
```

Comment:
```suggestion
    alcohol_degree: Column[float] = column(sa.Numeric(4, 2, asdecimal=False))
```

(voir mon com plus bas)